.vscode/settings.json

```diff
@@ -2,6 +2,9 @@
   "cSpell.words": ["blockaid", "lavamoat"],
   "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
   "editor.tabSize": 2,
+  "files.associations": {
+    "app/*.html": "ejs"
```

Comment:
Tells VSCode that `app/*.html` files are to be treated as EJS template files (it gracefully falls back to HTML if the EJS syntax plugin isn't installed)